### PR TITLE
#157110844 Ft Add Dropsets Support 

### PR DESCRIPTION
**Pull request by mutoromike**
**What does this PR do?**

This pull request adds a drop sets functionality, where a user is able to reduce the weights used in a specific exercise during a workout while still performing the same repetitions. The weight reduction is uniform, which is calculated in terms of a percentage. 

screen shot 2018-05-30 at 10 26 33

**What are specific Pivotal Tracker stories?**
Add support for drop sets
ID: #157110844

### CODE REVIEW

**Review by Kalela — APPROVED** [review]
LGTM

**Review by FelixWambiri — APPROVED** [review]
Nice implementation especially on testing your work and increasing the test coverage

File: wger/core/tests/base_testcase.py [hunk 1]
```diff
@@ -561,14 +561,14 @@ def test_access_anonymous(self):
         self.user_logout()
         self.access(fail=self.anonymous_fail)
 
-    # def test_access_authorized(self):
-    #     '''
-    #     Tests accessing the URL as the authorized users
-    #     '''
+    def test_access_authorized(self):
```
Comment (meshnesh) [inline wger/core/tests/base_testcase.py:564]: Good implementation of the tests.

**Review by meshnesh — APPROVED** [review]
good working adding tests to cover your story and completing it in the process.

**Review by betsybeth — APPROVED** [review]

**Pull request merged**